part4/SpaceSciencePython_part4.py

```diff
@@ -110,6 +110,10 @@
 from matplotlib import pyplot as plt
 import matplotlib.dates as matpl_dates
 
+# Convert UTC to the matplotlib date format
+INNER_SOLSYS_DF.loc[:, 'UTC'] = \
+    INNER_SOLSYS_DF['UTC'].apply(lambda x: matpl_dates.date2num(x))
```

Comment:
I checked this issue with matplotlib 3.2.1 and it appears that this issue does not occur. Other issues are approved